docs/guides/build-notifications.rst

```diff
@@ -35,9 +35,11 @@ The project name, slug and its details for the build that failed will be sent as
             "slug": "rtd",
             "build": {
                 "id": 6321373,
+                "commit": "e8dd17a3f1627dd206d721e4be08ae6766fda40",
+                "state": "finished",
                 "success": false,
                 "date": "2017-02-15 20:35:54"
             }
        }
 
-You should now get notified on your webhook when your builds fail!
+You should now get notified on your webhook when your builds start and finish (failure/success)!
```

Comment:
It seems this is true only for webhook notification but not for email notification. I think that's good because we don't want to spam user's email inbox but this should be clearer in the docs.

Reply:
This is the webhook section of the notification docs so I only wrote only about webhook. Email section only says that we will send emails only if the build fails. 
https://github.com/readthedocs/readthedocs.org/blob/19e16c43b329e23eec36edfa18166b9ac63397c9/docs/guides/build-notifications.rst#using-email